lib/wallaroo/core/sink/connector_sink/connector_sink_builder.pony

```diff
@@ -147,7 +147,10 @@ class val ConnectorSinkBuilder
       ("ConnectorSinkBuilder: Connecting to sink at " + _host + ":" + _service + "\n")
       .cstring())
 
-    let id: RoutingId = RoutingIdGenerator()
+    let id: RoutingId = try DeterministicSourceIdGenerator(
```

Comment:
In this case, I think we should rename this generator. Maybe `RoutingIdFromStringGenerator`?